packages/react/src/DevMode.tsx

```diff
@@ -26,31 +26,415 @@ export const DevMode = ({ position = 'bottomRight' }: DevModeProps) => {
 const ModeSubscriber = ({ position }: { position: keyof typeof Position }) => {
   const [isHover, setIsHover] = useState(false)
   const devMode = useDevModeObserve()
-  if (devMode == null) {
-    return null
+  if (process.env.NODE_ENV !== 'production' && devMode != null) {
+    return (
+      <svg
+        role={`Suspensive.DevMode-${devMode.is ? 'on' : 'off'}`}
+        onClick={devMode.is ? devMode.off : devMode.on}
+        onMouseEnter={() => setIsHover(true)}
+        onMouseLeave={() => setIsHover(false)}
```

Comment:
I'm wondering if we need to call `useState` to control hover state.

Since we can't control the hover state from **inline CSS**, I guess it's true that we can only give the hover effect by controlling the state via `useState` at this point.

But in essence, I don't see **how adding a hover state to this UI can provide any value to the user**. 

Personally, I think a hover effect is meant **to encourage clicks**, but since `DevMode` is something that the user **will do on their own when they need to**

Personally, I think the hover effect is used to encourage clicking, but 'DevMode' assumes that the user will press it when they need to.

To summarize, **do we need to use `useState` to control the hover state?** is my opinion.

Reply:
I found there is no svg or no useState code in built bundle. you can check it in websites/visualization with DevMode.

<img width="449" alt="image" src="https://github.com/suspensive/react/assets/61593290/bcc7af42-678b-453b-b454-e03319f14dd9">

Then isn't it better to support better UX in development mode?

Reply:
Got it! Thanks for the confirmation!